### ISSUE TITLE
Environment variable not being saved after copy/paste

### ISSUE THREAD

**Issue by marcellorvalle** [getinsomnia/insomnia#2383]
**Describe the bug**
The value of the environment variables are not saved if you just copy/paste and don't type into the json area.

**To Reproduce**
1. Go to 'Manage Environments'
2. Create a new pair variableA/valueA 
3. Click 'Done'
4. Go back to 'Manage Environments'
5. Copy some valueB (from notepad for example)
6. Paste as valueB over valueA  (just paste, do not type anything)
7. Click 'Done'
8. Go back to 'Manage Environments'
9. See that the value of variableA is still valueA

**Expected behavior**
The value of variableA should be valueB.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Installation Method: update and download
 - App Version: 2020.03.0

**Comment by MBerbert** [comment]
I'm having the same problem when pasting the token into the Bearer.

**Desktop (please complete the following information):**

- OS: Windows 10
- Installation Method: update and download
- App Version: 2020.03.0

**Comment by marcellorvalle** [comment]
A workaround is to type something and just delete it before saving it.

**Comment by kjaebker** [comment]
This seems to be happening across the whole app. I am also experiencing it in the Query Variables section for GraphQL requests. After pasting in a new value, it keeps the old value even though the display has been updated. The workaround from @marcellorvalle does work here as well.

**Comment by alexislopes** [comment]
As @marcellorvalle mentioned, I followed this steps:

- click the field and type something;
- click out of the field;
- click the field again and paste your content right after the content typed;
- click out of the field and it should work;
- then just clean the previosly typed content;

I tried to paste right before the typed chracters, same fail.

**Comment by xyphr** [comment]
I'm experiencing the same issue. Pasting anything in the environment doesn't stick, only if I manually type something then paste then it gets saved.

**Comment by nijikokun** [comment]
Duplicate of #2377

**Comment by stefer** [comment]
This is still an issue, even though #2377 is closed since long.
In my case I have a a reponse tag in the request which was not saved when closing the Environments page.
I had a look in the insomnia.Environment.db file and there where lines like the one below.
However, when just changing the response tag to set request etc, a new line was added, but without the request reference.

When I added the property "blabla" in the environment it started working again.

`
{"_id":"env_49bfc291c9dd4ced8177a0408178de94","type":"Environment","parentId":"env_09d97291147b89c5e9ca04bc139c1e48e1d22dc8","modified":1673618345835,"created":1673592277671,"name":"test","data":{"scheme":"https","base_path":"","host":"api-vera-test.collectordev.se","bearerToken":"{% response 'body', 'req_42d314e4f2a74e13845f92ff35f50b75', 'b64::JC5hY2Nlc3NfdG9rZW4=::46b', 'never', 60 %}","tokenUrl":"XXX","blabla":"aa"},"dataPropertyOrder":{"&":["scheme","base_path","host","bearerToken","tokenUrl","blabla"]},"color":null,"isPrivate":true,"metaSortKey":1673591907088}
`
Version: Insomnia 2022.7.2
Build date: 1/12/2023
OS: Windows_NT x64 10.0.19044
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: x64
